### PR TITLE
Add class event calendar

### PR DESCRIPTION
**Pull request by Ybbet**
Hello,
This pull request allows you to add a class to the tr of the event sought in customEvents. You must enter the class with the "eventClass" option.

### CODE REVIEW

**Comment by rejas** [comment]
Thx for the PR. Could you run prettier over the code and commit again?

File: modules/default/calendar/calendar.js [hunk 1]
```diff
@@ -337,6 +337,9 @@ Module.register("calendar", {
 							}
 							break;
 						}
+						if (typeof this.config.customEvents[ev].eventClass !== "undefined" && this.config.customEvents[ev].eventClass !== "") {
```
Comment (rejas) [inline modules/default/calendar/calendar.js:340]: you should add info about eventClass to line 44 (and later in a seperate PR to the documentation repo)

File: modules/default/calendar/calendar.js [hunk 2]
```diff
@@ -337,6 +337,9 @@ Module.register("calendar", {
...
+							eventWrapper.className += ` ${this.config.customEvents[ev].eventClass}`;
```
Comment (rejas) [inline modules/default/calendar/calendar.js:341]: Are you interested in writing a test for this? sjould be in there: https://github.com/MichMich/MagicMirror/blob/abe5c08a52ffee6b37c30c1f3160e130f0e966a2/tests/e2e/modules/calendar_spec.js#L45

**Review by rejas — CHANGES_REQUESTED** [review]

File: modules/default/calendar/calendar.js (hunk 2 above)
Comment (Ybbet) [reply modules/default/calendar/calendar.js:341]: Done ;-)

File: modules/default/calendar/calendar.js (hunk 1 above)
Comment (Ybbet) [reply modules/default/calendar/calendar.js:340]: Done ;-)

**Review by rejas — APPROVED** [review]

**Comment by rejas** [comment]
Thx for the PR. Now can you create a PR for this to update the Documentation in the MM-Doc repo?

**Comment by Ybbet** [comment]
You welcome. I will create it tomorrow. 😊

**Pull request merged**